api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -849,27 +835,6 @@ func (in *RuntimeProperties) DeepCopy() *RuntimeProperties {
 	return out
 }
 
-// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in RuntimePropertiesMap) DeepCopyInto(out *RuntimePropertiesMap) {
```

Comment:
Same here. If we remove these types, we might run into problems when parsing the k8s API data.

Reply:
`RuntimePropertiesMap` is just a interface now. No need to define it as a `struct` by itself.
https://github.com/vaibhavjainwiz/kogito-cloud-operator/blob/e48493340cab7c1f9754cab846ecc6ee45571a2b/api/kogitoinfra_types.go#L96